docs/network/interfaces_and_networks.md

```diff
@@ -509,6 +509,78 @@ spec:
 > **Note:** Placement on dedicated CPUs can only be achieved if the Kubernetes CPU manager is running on the SR-IOV capable workers.
 > For further details please refer to the [dedicated cpu resources documentation](../compute/dedicated-cpu_resources.md/).
 
+### Link State Management
```

Comment:
Continuing adding stuff in this file is problematic.

I want to re-organize the documentation we have and start splitting it into smaller pieces.
For now, I do not know what is the correct separation.
So I am good to place it here for now and suggest a better location later.